OSDOCS-1720 bug fixes for 4-6 branch only

bobfuru opened the pull request:
Follow up to https://github.com/openshift/openshift-docs/pull/28034 and to this [comment](https://github.com/openshift/openshift-docs/pull/28034#issuecomment-749778592) that brings OCP 4.6 docs up to date for: 
- CSI snapshot controller namespace, as reported in https://bugzilla.redhat.com/show_bug.cgi?id=1906288
- Use of snapshot apiVersion `snapshot.storage.k8s.io/v1beta1`
- Adding `modules/persistent-storage-csi-drivers-supported.adoc` using the drivers supported up to OCP 4.6

bobfuru commented:
@duanwei33 PTAL for QE

duanwei33 commented on modules/persistent-storage-csi-drivers-supported.adoc:
```diff
@@ -0,0 +1,28 @@
+|CSI driver  |CSI volume snapshots  |CSI cloning  |CSI resize |CSI ephemeral volumes
+
+|AWS EBS (Tech Preview) | ✅ | - | ✅ | -
+|OpenStack Cinder | ✅ | ✅ | ✅ | ✅
```
We don't have Cinder csi driver in OCP4.6.

bobfuru replied:
Removed Cinder

duanwei33 commented:
@bobfuru 
Is there another bz to track PersistentVolume and PersistentVolumeClaim as not a CRD?

Similar to the PersistentVolume **CRD**, the VolumeSnapshotContent CRD is a cluster resource that points to a real snapshot in the storage back end.
Similar to the PersistentVolumeClaim **CRD**, the VolumeSnapshot CRD defines a developer request for a snapshot.

bobfuru commented:
@duanwei33 - Removed Cinder, let me know if this looks good now, thanks!

bobfuru commented:
> @bobfuru
> Is there another bz to track PersistentVolume and PersistentVolumeClaim as not a CRD?
> 
> Similar to the PersistentVolume **CRD**, the VolumeSnapshotContent CRD is a cluster resource that points to a real snapshot in the storage back end.
> Similar to the PersistentVolumeClaim **CRD**, the VolumeSnapshot CRD defines a developer request for a snapshot.

Good catch on this. I've updated this PR and the 4.5 PR (https://github.com/openshift/openshift-docs/pull/28330) to conditionally reflect the additional bug fixes that were also introduced in 4.7 (https://github.com/openshift/openshift-docs/pull/28034).

duanwei33 commented:
/lgtm

bobfuru commented:
@openshift/team-documentation PTAL

sfortner-RH commented on modules/persistent-storage-csi-drivers-supported.adoc:
```diff
@@ -0,0 +1,26 @@
+[id="csi-drivers-supported_{context}"]
+= CSI drivers supported by {product-title}
+
+{product-title} installs certain CSI drivers by default, giving users storage options that are not possible with in-tree volume plug-ins.
```
Do we need to spell out the first instance of CSI here for clarity. There doesn't seem to be a consolidated viewpoint on doing this in OCP, but if a user reaches this doc from Google will they know what that acronym stands for? I always spell out the first instance of an acronym in all of my topics, even if they are nested together in the same assembly, and then proceed with acronym throughout for this reason. But it's also a hard ingrained habit from past roles too. It's fine as it, just some thoughts :)

bobfuru replied:
Yes, I'm in total agreement. I try to spell out on first instance at the module level but missed this one, now fixed. Thanks!

sfortner-RH reviewed (approved):
lgtm!
